sale_coupon_order_line_link/__manifest__.py

```diff
@@ -4,11 +4,11 @@
     "name": "Link coupons to order lines",
     "summary": "Adds a link between coupons and their generated order lines for easing "
     "tracking",
-    "version": "13.0.1.2.0",
+    "version": "14.0.1.0.0",
     "development_status": "Production/Stable",
     "category": "Sale",
     "website": "https://github.com/OCA/sale-promotion",
-    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "author": "Ilyas, Ooops404, Tecnativa, Odoo Community Association (OCA)",
```

Comment:
I'm afraid a migration of this scope doesn't grant you co-authorship. You can put yourself on contributors though

Reply:
@pedrobaeza is it ok now ?